src/main/java/core/basesyntax/shop/service/Operations.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.shop.service;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+public enum Operations {
```

Comment:
```suggestion
public enum Operation {
```
enum names should better be singular